lib/revm_consistency_checker/src/bytecode_hash.rs

```diff
@@ -0,0 +1,95 @@
+use alloy::primitives::B256;
```

Comment:
are unsafes necesarry in this file? This is the first place in the codebase where we introduce it. Is there a way around it? what's the trade-off?